builds/vs2015/PlayerLib.vcxproj

```diff
@@ -752,4 +756,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Did you use MSVC to save this file? If not, I would like to keep the EOF inebreak.